.codesandbox/ci.json

```
@@ -1,11 +1,12 @@
 {
-  "buildCommand": "build:min",
+  "buildCommand": "build --to @fluentui/react --to @fluentui/react-components --to @fluentui/react-northstar",
```

Reverting this for now
```suggestion
  "buildCommand": "build:min",
```